samples/tab-personal-quickstart/js/src/components/__tests__/Tab.spec.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import sinon from "sinon";
+import proxyquire from "proxyquire";
+import { app } from "@microsoft/teams-js";
+
+const appStub = sinon
```

Comment:
This is the actual interesting change in this PR. It mocks out the Teams JS library